File: articles/distribution-shift-and-model-decay/go/decay_risk.go
package main

import "fmt"

func main() {
	inputDrift := 0.31
	labelDrift := 0.16
	performanceDecay := 0.10
	calibrationGap := 0.14
	subgroupGap := 0.15
	overrideRate := 0.11
	score := (inputDrift + labelDrift + performanceDecay + calibrationGap + subgroupGap + overrideRate) / 6.0
	fmt.Printf("decay_risk_score=%.4f\n", score)
}

File: articles/distribution-shift-and-model-decay/typescript/decay_risk.ts
const inputDrift = 0.31;
const labelDrift = 0.16;
const performanceDecay = 0.10;
const calibrationGap = 0.14;
const subgroupGap = 0.15;
const overrideRate = 0.11;
const score = (inputDrift + labelDrift + performanceDecay + calibrationGap + subgroupGap + overrideRate) / 6.0;
console.log(`decay_risk_score=${score.toFixed(4)}`);
